doc/code/qml_qaoa.rst

```diff
@@ -96,7 +96,12 @@ QAOA Layers
 Cycle Optimization
 ------------------
 
-.. automodapi:: pennylane.qaoa.cycle
-    :no-heading:
-    :no-inheritance-diagram:
-    :no-inherited-members:
+The :mod:`~.cycle` module is available for additional functionality related to the maximum-weighted
```

Comment:
Looks good! The only question I have is whether `cycle_mixer()` belongs in `qaoa.cycle.py` or `qaoa.mixers.py`?